packages/wasmfs/rollup.config.js

```diff
@@ -1,65 +0,0 @@
-// Rollup Config for the Wasm File System Example
-
```

Comment:
So we are removing rollup for this package? How does it interact with the other rollup and watching now? 🤔

Reply:
In fact, how is this project built / used? 🤔

Reply:
We are using the inner build system of memfs (tsc directly)